CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## <<DATE>> libunftp <<VERSION
```

Comment:
Just call it `## Upcoming` then it reads nice to the user too. That is if we're gonna add more till we release.